generator/generator_tests/road_access_test.cpp

```diff
@@ -0,0 +1,124 @@
+#include "testing/testing.hpp"
+
+#include "generator/generator_tests_support/routing_helpers.hpp"
+#include "generator/generator_tests_support/test_feature.hpp"
+#include "generator/generator_tests_support/test_mwm_builder.hpp"
+
+#include "generator/road_access_generator.hpp"
+
+#include "routing/road_access_serialization.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/file_name_utils.hpp"
+
+#include "platform/platform_tests_support/scoped_dir.hpp"
+#include "platform/platform_tests_support/scoped_file.hpp"
+
+#include "platform/country_file.hpp"
+#include "platform/platform.hpp"
+
+#include "base/logging.hpp"
+
+#include <string>
+
+using namespace feature;
+using namespace generator;
+using namespace platform::tests_support;
+using namespace platform;
+using namespace routing;
+using std::string;
+
+namespace
+{
+string const kTestDir = "road_access_generation_test";
+string const kTestMwm = "test";
+string const kRoadAccessFilename = "road_access_in_osm_ids.csv";
+string const kOsmIdsToFeatureIdsName = "osm_ids_to_feature_ids" OSM2FEATURE_FILE_EXTENSION;
+
+void BuildEmptyMwm(LocalCountryFile & country)
+{
+  generator::tests_support::TestMwmBuilder builder(country, feature::DataHeader::country);
+}
+
+void LoadRoadAccess(string const & mwmFilePath, RoadAccess & roadAccess)
+{
+  FilesContainerR const cont(mwmFilePath);
+  TEST(cont.IsExist(ROAD_ACCESS_FILE_TAG), ());
+
+  try
+  {
+    FilesContainerR::TReader const reader = cont.GetReader(ROAD_ACCESS_FILE_TAG);
+    ReaderSource<FilesContainerR::TReader> src(reader);
+
+    RoadAccessSerializer::Deserialize(src, roadAccess);
+  }
+  catch (Reader::OpenException const & e)
+  {
+    LOG(LERROR, ("Error while reading", ROAD_ACCESS_FILE_TAG, "section.", e.Msg()));
```

Comment:
Nice catch.
LCRITICAL would be enough even in tests, but not LERROR.
Changed to a more strict TEST(false, ...).